config.go

```diff
@@ -106,6 +108,27 @@ func (c *Config) Discover() error {
 		}
 	}
 
+	// Finally, we'll fill in any internal plugins that haven't already been
+	// discovered on disk.
```

Comment:
Can you point to where the plugin API version is specified? I can probably put a version check in that part of the code. I don't think packer has that feature.